services/userService.js

```diff
@@ -0,0 +1,18 @@
+import bcrypt from 'bcryptjs';
+import { userDao } from '../models';
+
+const createUser = async (userAllInfo) => {
+  const { idForLogin, password } = userAllInfo;
+
+  const existedId = await userDao.getUserId(idForLogin);
+  if (existedId.length) {
```

Comment:
DB에서 일치하는 아이디 없으면 falsy한 값을 리턴하니까
```suggestion
  if (existedId) {
```
로 쓰는게 좋을 것 같아요.